internal/querynodev2/segments/index_attr_cache.go

```diff
@@ -61,7 +62,7 @@ func (c *IndexAttrCache) GetIndexResourceUsage(indexInfo *querypb.FieldIndexInfo
 		return 0, 0, fmt.Errorf("index type not exist in index params")
 	}
 	if indexType == indexparamcheck.IndexDISKANN {
-		neededMemSize := indexInfo.IndexSize / UsedDiskMemoryRatio
+		neededMemSize := int64(float64(indexInfo.IndexSize) * paramtable.Get().QueryNodeCfg.DiskIndexMemoryFactor.GetAsFloat())
```

Comment:
DISKANN has not been changed. Why need this update?

Reply:
ran into oom when loading disk index under the case of LRU